paragraph/scheduling/graph_scheduler.cc

```diff
@@ -47,17 +48,31 @@ shim::StatusOr<std::unique_ptr<GraphScheduler>> GraphScheduler::Create(
           InstructionFsm(scheduler.get(), instruction.get()));
     }
   }
+  return scheduler;
+}
+
+absl::Status GraphScheduler::Init(double seconds) {
```

Comment:
"seconds" is a unit, but the name doesn't say what it represents. Use "current_time_secs".